Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuerySecuredInterfaceTest/pom.xml

```diff
@@ -7,7 +7,7 @@
     </parent>
 
     <artifactId>EntityDocQuerySecuredInterfaceTest</artifactId>
-    <name>Maven 3 soapUI EntityDocQuerySecuredInterfaceTest</name>
+    <name>Maven 3 soapUI EntityDocQuerySecuredInterfaceTest-Standard</name>
```

Comment:
This is for easy read when looking at failed nightly job. This won't affect soapUI project